validator/client/log.go

```diff
@@ -45,16 +45,23 @@ func (v *validator) LogNextDutyTimeLeft(slot uint64) error {
 	}
 
 	var nextDutySlot uint64
-	var role string
+	attestingCount := 0
+	proposingCount := 0
 	for _, duty := range v.duties.CurrentEpochDuties {
-		if duty.AttesterSlot > slot && (nextDutySlot > duty.AttesterSlot || nextDutySlot == 0) {
+		if duty.AttesterSlot == nextDutySlot {
+			attestingCount += 1
+		} else if duty.AttesterSlot > slot && (nextDutySlot > duty.AttesterSlot || nextDutySlot == 0) {
 			nextDutySlot = duty.AttesterSlot
-			role = "attester"
+			attestingCount = 1
```

Comment:
```suggestion
			attestingCount = 1
```

it doesn't seem right to set `attestingCount` to 1 here